orchagent/intfsorch.cpp

```diff
@@ -389,7 +403,8 @@ set<IpPrefix> IntfsOrch:: getSubnetRoutes()
     return subnet_routes;
 }
 
-bool IntfsOrch::setIntf(const string& alias, sai_object_id_t vrf_id, const IpPrefix *ip_prefix, const bool adminUp, const uint32_t mtu)
+bool IntfsOrch::setIntf(const string& alias, const string& vrf_name, sai_object_id_t vrf_id, const IpPrefix *ip_prefix, 
```

Comment:
Lets discuss this - Why we need to pass the vrf_name to this function.

Reply:
This helps to ignore the HW programming of the nbr with in-band-intf which is part of the mgmt VRF.